.babelrc

```diff
@@ -16,5 +16,24 @@
         "loose": true
       }
     ]
-  ]
+  ],
+
+  "env": {
+    "development": {
+      "plugins": ["external-helpers"]
+    },
+
+    "production": {
+      "plugins": [
+        "external-helpers",
+        "transform-react-inline-elements",
+        "transform-react-constant-elements",
```

Comment:
I struggle to understand what's the benefit of this pass. If the generated code is minified, the constant should be inlined into the function if the minifier support inlining of invariants:
```js
// Before:
const Hr = () => {
  return <hr className="hr" />;
}
// After:
const _ref = <hr className="hr" />;
const Hr = () => {
  return _ref;
};
// After minification:
const Hr = () => {
  return <hr className="hr" />;
};
```